cirq-core/cirq/circuits/moment_test.py

```diff
@@ -693,6 +693,18 @@ def test_commutes():
     assert not cirq.commutes(moment, cirq.X(c))
 
 
+def test_commutes_multiqubit_gates():
+    a = cirq.NamedQubit('a')
```

Comment:
this test case is very simple and doesn't cover the more complex cases ... see above